sn_registers/src/register.rs

```diff
@@ -27,7 +27,7 @@ const MAX_REG_NUM_ENTRIES: u16 = 1024;
 #[derive(Clone, Eq, PartialEq, PartialOrd, Hash, Serialize, Deserialize, Debug)]
 pub struct Register {
     /// Owner of the Register
-    owner: User,
+    owner: PublicKey,
```

Comment:
Changed it so the owner of a Register is a `PublicKey`, as it cannot be `User::Anyone`